docs/intro.md

```diff
@@ -20,27 +20,31 @@ The full node daemon, `TBD` is written in Rust and based on [substrate](https://
 					     	!  !
 						    !  !
 						    !  !
-                _(_)_       !  !                   _(_)_ 
-    @@@@       (_)@(_)     (____)      @@@@       (_)@(_) 
-   @@()@@ wWWWw  (_)\      (____)     @@()@@ wWWWw  (_)\ 
+                _(_)_       !  !                   _(_)_
+    @@@@       (_)@(_)     (____)      @@@@       (_)@(_)
+   @@()@@ wWWWw  (_)\      (____)     @@()@@ wWWWw  (_)\
     @@@@  (___)     \|/    (____)      @@@@  (___)     \|/
-     /      Y       \|     (____)       /      Y       \| 
-  \ |     \ |/       | /    !  !     \ |     \ |/       | / 
-  \\|//   \\|///  \\\|//    !  !     \\|//   \\|///  \\\|// 
-^^^^^^^^^^^^^^^^^^^^^^^^^   \__/   ^^^^^^^^^^^^^^^^^^^^^^^^^^ 
+     /      Y       \|     (____)       /      Y       \|
+  \ |     \ |/       | /    !  !     \ |     \ |/       | /
+  \\|//   \\|///  \\\|//    !  !     \\|//   \\|///  \\\|//
+^^^^^^^^^^^^^^^^^^^^^^^^^   \__/   ^^^^^^^^^^^^^^^^^^^^^^^^^^
 ```
+
 ## Source code
 The latest source code is available on [Github](https://github.com/radicle-dev) under the [GPL v3.0](https://opensource.org/licenses/GPL-3.0) license.
 
-## Community & IRC
-Radicle is an open platform owned by the commons. You can participate or follow the discussions in our [community forum](https://radicle.community/). and you can follow us on Twitter [@radicle_xyz](https://twitter.com/radicle_xyz).
-
-A community IRC channel exists on Freenode in irc.freenode.net:#radicle.
 
 ## Tokens
 
 The integrity of our canonical registry is secured through an incentivized [proof-of-work (PoW)](https://en.wikipedia.org/wiki/Proof_of_work) based consensus scheme. The Registry uses a utility token to support its native functions such as project registration, checkpointing etc. (`TBD` better explanation).
 
 We strive to achieve network ownership by the [FOSS](https://en.wikipedia.org/wiki/Foss) community from [genesis](https://en.bitcoin.it/wiki/Genesis_block).
 
----
\ No newline at end of file
+## Get in touch
+
+We would love to hear from you!
+There are several ways to get in touch with us:
+
+- Participate or follow the discussions in our [community forum](https://radicle.community/)
+- Follow us on Twitter [@radicle_xyz](https://twitter.com/radicle_xyz).
+- Join our <a href="https://webchat.freenode.net/#radicle" target="_blank">Comunity IRC channel</a>
```

Comment:
Is there a reason this opens in a new tab while the other links do not?

Reply:
The others should too since they take the user to another location. I just copy-pasted them from above. Fixing :running_man: